distribution/ddf-common/src/main/resources/security/configurations.policy

```diff
@@ -42,3 +42,26 @@ grant codeBase "file:/catalog-core-directorymonitor/org.apache.camel.camel-core/
     // permission java.io.FilePermission "/test/some/monitored/directory", "read";
     // permission java.io.FilePermission "/test/some/monitored/directory${/}-", "read, write";
 }
+
+//                        URL Resource Reader Permissions
+// Adding required permissions for the URL Resource Reader to retrieve file resources.
+//
+//
+// Configuring URL Resource Reader requires adding read and write permissions to the directory being monitored.
+// For the following 2 permissions are required:
+//   1. permission java.io.FilePermission "<DIRECTORY_PATH>", "read";
+//   2. permission java.io.FilePermission "<DIRECTORY_PATH>${/}-", "read";
+// Replacing <DIRECTORY_PATH> with the path of the directory containing file resources.
+//
+// Line 1 gives the URL Resource Reader the permissions to read from the directory. Line 2 gives the
+// Resource Reader the permissions to recursively read from the directory path, specified
+// by the directory path's suffix "${/}-".
+//
+//
+grant codeBase "file:/catalog-core-urlresourcereader" {
```

Comment:
❗️ We might need to add `org.apache.tika.core` here for when this path is hit https://github.com/codice/ddf/blob/2.13.x/catalog/core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java#L453. I can supply a file to test this path if needed.

Reply:
@peterhuffer I'll take the sample file if you have it handy.